_data/navbar.yml

```diff
@@ -31,6 +31,7 @@
       - xen-api/metrics.md
       - xen-api/memory.md
       - xen-api/xencenter.md
+        # - profiling/coverage.md
```

Comment:
I wasn't sure where to put it. I had it here first before I turned it into a design doc. I'm still not sure.